### DISCUSSION TITLE
v18.0

### DISCUSSION THREAD

**Discussion by carloscabanero** [blinksh/blink#2023] [Announcements]
Next Blink version will align with new iOS - iPadOS and macOS release.

**Comment by carloscabanero** [comment]
IMPORTANT: If using iOS18 beta, please make sure to update your devices to the latest version. If using iOS17, everything should still work as expected.

Build 935 / AppStore Build 936

* Added support for light and dark icons on the new iOS18 home screen. #2048. Check out [how to configure Dark/Light icons](https://support.apple.com/guide/iphone/customize-apps-and-widgets-on-the-home-screen-iph385473442/18.0/ios/18.0)
* Adjusted UI for new iPhone 16 seres.
* On iOS18, fixed crashes on startup and when running certain commands. Thanks to TestFlight reports.
* On iOS18, resolved issues with unresponsive sessions when switching tabs after restoring the app from the background. #2047
* Updated OpenSSH and other libraries for iOS18 compatibility.
* Files.app and SFTP/SCP command issues when overwriting files to remote destinations. #2050
* Fixed for random mosh sessions closing without storing state.
* OpenSSH, Vim, xxd libraries updated to work on iOS18.
* Released from new build system, tested everything is in place.
* iOS17 and iPadOS17 are now the minimum version required to run Blink Shell.

**Comment by larme** [comment]
Hi @carloscabanero, do we still plan to release the new MacOS version?

**Reply by carloscabanero** [reply]
I will get back to you on that in the proper thread #1420

**Comment by carloscabanero** [comment]
Version 18.0.1 build 937 (Community) and 940 (AppStore):
- Fixes keyboard repeat on accentuated characters. #1965 
Kudos to @yury

Known Issues:
- Fixing Files.app is our number one priority #2051, #1717
- Mosh server has issues in macOS Sequoia due to the new Local Network Privacy policies. https://github.com/mobile-shell/mosh/issues/1254#issuecomment-2405102497
